nntrainer/layers/layer_node.cpp

```diff
@@ -116,7 +116,12 @@ int LayerNode::setProperty(std::vector<std::string> properties) {
     auto &ac = nntrainer::AppContext::Global();
     std::unique_ptr<nntrainer::Layer> dlayer =
       ac.createObject<nntrainer::Layer>(TimeDistLayer::type);
-    dynamic_cast<TimeDistLayer *>(dlayer.get())->setDistLayer(std::move(layer));
+    if (dlayer.get() == nullptr)
```

Comment:
Is there any change dlayer ( ac.createObject()) is nullptr? if it is, definitely createObject need to throw.

Reply:
this is done to resolve coverity issue.